wallet/wallet.go

```diff
@@ -206,6 +213,98 @@ func (w *SingleAddressWallet) FundTransaction(cs consensus.State, txn *types.Tra
 	return toSign, nil
 }
 
+// Split returns a transaction that splits the wallet in the given number of
+// outputs with given amount. The transaction is funded but not signed.
+//
+// NOTE: split needs to use a minimal set of inputs and therefore does not reuse
+// the fund logic which randomizes the unspent transaction outputs used to fund
+// the transaction
+func (w *SingleAddressWallet) Split(cs consensus.State, outputs int, amount, feePerByte types.Currency, pool []types.Transaction) (types.Transaction, []types.OutputID, error) {
```

Comment:
I think in `renterd` it does make sense to have a `Split` handler due to the selection algorithm being different than we use in `fund` and also because we need `w.Address()`. Could be an argument but seeing as `Split` will always need the wallet address it's safer to have the method on the wallet imo.